plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/CountingAccessHiveMetastore.java

```diff
@@ -113,7 +115,8 @@ public List<String> getTablesWithParameter(String databaseName, String parameter
     @Override
     public List<String> getAllViews(String databaseName)
     {
-        throw new UnsupportedOperationException();
+        methodInvocations.add(Methods.GET_ALL_VIEWS_WITH_PARAMETER);
+        return delegate.getAllViews(databaseName);
```

Comment:
why `_WITH_PARAMETER`? shouldn't this be `GET_ALL_VIEWS`?

Reply:
`GET_ALL_VIEWS_FROM_DATABASE` then

Reply:
Renamed